h/views/admin/users.py

```diff
@@ -42,6 +42,23 @@ def users_index(request):
         if user is None:
             user = models.User.get_by_email(request.db, username, authority)
 
+    if (user is not None) and user.deleted:
```

Comment:
`if user`?

Reply:
Yeah :man_shrugging: I was just matching the `if user is None` above.

I'm actually not totally sure I agree with Pylint's `if foo` rather than `if foo is not None` all the time. There's a difference between `foo` being a falsey value and `foo` being `None`. `None` is being used as a special sentinel value here to mean "the DB query found no matching rows". I find myself wanting to actually say the more accurate `is not None` more and more often these days. In this particular instance it wouldn't make any difference, at least not right now, because `User` objects are never falsey.